__tests__/app.test.js

```diff
@@ -82,8 +82,70 @@ describe("get api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        const { apiDetails } = response.body;
-        expect(apiDetails).toEqual(endpoints);
+        const { apiData } = response.body;
+        expect(apiData).toEqual(endpoints);
+      });
+  });
+});
+
+describe("get articles by id", () => {
+  test("should respond with an object with correct keys ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(typeof article).toBe("object");
+        expect(Array.isArray(article)).toBe(false);
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
```

Comment:
Here we want to explicitly check the Id is 1 - in order to ensure we have the correct article back